src/components/ItemsList.js

```diff
@@ -104,12 +106,28 @@ const ItemsList = () => {
                   .toLowerCase()
                   .includes(searchTerm.toLowerCase()),
               )
+              .sort((a, b) =>
+                a.daysLeftForNextPurchase > b.daysLeftForNextPurchase
+                  ? 1
+                  : a.daysLeftForNextPurchase === b.daysLeftForNextPurchase
+                  ? a.shoppingListItemName > b.shoppingListItemName
+                    ? 1
+                    : -1
+                  : -1,
+              )
```

Comment:
I have totally failed to understand and follow the logic in this ternary operator. I think it would be more readable to use `if else` than a chain of ternary operators.

Reply:
Ooh sorry about that. we used it because its shorter and also we indented it that way for better understanding. I will discus with my partner on how best to go about it and probably get inputs from the mentors.

Reply:
Thanks for the suggestion Joseph. You could be right! Since I knew the long version of it, this ternary was easy for me to understand. But maybe it wouldn't have been if I was not the one working on this code. Yes Chiamaka, let's wait for the input of the mentors for this one.